specification/network/resource-manager/Microsoft.Network/stable/2023-06-01/firewallPolicy.json

```diff
@@ -1708,7 +1708,11 @@
       "properties": {
         "mode": {
           "$ref": "#/definitions/FirewallPolicyIntrusionDetectionStateOptions",
-          "description": "Intrusion detection general state."
+          "description": "Intrusion detection general state. When attached to a parent policy, the firewall's effective IDPS mode is the stricter mode of the two."
+        },
+        "profile": {
```

Comment:
Isn't this breaking change for existing customers, who don't know that now they have to send this new property in their requests?

Reply:
No, this property is optional

Reply:
However, the existing customers who are using this stable API version, will not send this property, as they are not expecting it in the same API version. Why is it being added in same API version, not in new API Version?